susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1116,6 +1116,18 @@
         'BASECHANNEL' : 'centos8-x86_64', 'PKGLIST' : RES8,
         'DEST' : '/srv/www/htdocs/pub/repositories/centos/8/bootstrap/'
     },
+    'oracle-6-x86_64' : {
+        'BASECHANNEL' : 'oraclelinux6-x86_64', 'PKGLIST' : RES6,
+        'DEST' : '/srv/www/htdocs/pub/repositories/oracle/6/bootstrap/'
+    },
+    'oracle-7-x86_64' : {
+        'BASECHANNEL' : 'oraclelinux7-x86_64', 'PKGLIST' : RES7,
+        'DEST' : '/srv/www/htdocs/pub/repositories/oracle/7/bootstrap/'
+    },
+    'oracle-8-x86_64' : {
+        'BASECHANNEL' : 'oraclelinux8-x86_64', 'PKGLIST' : RES8,
+        'DEST' : '/srv/www/htdocs/pub/repositories/oracle/8/bootstrap/'
+    },
```

Comment:
We have product ids for them and should make use of it.
For centos as well.

Reply:
Then I see two problems:
a) What to do about existing CentOS bootstrap entries? They are already created. Can we just append `-uyuni` for the existing entries and hope users will start using them?
b) I don't have the list of IDs, can you provide them? Or @lucidd? I'd need them for CentOS6/7/8 and Oracle6/7/8 base schannels.

What's the benefit of using PDIDs here vs BASECHANNEL if this forces us to duplicate entries?

Reply:
Ids are
C6i386 -10
C6x86_64 -11
C7x86_64 -12
C8x86_64 -13
OL7x86_64 -14
OL6x86_64 -15
OL6i386 -16
OL8x86_64 -17

Reply:
Clarified with @lucidd at rocket.chat, we don't need BETAPDIDs for CentOS and Oracle, as we'll control that from sumatoolbox.

Reply:
`C6i386` PDID : [-10, 1681], 'BETAPDID' : [2063]
`C6x86_64` PDID : [-11, 1682], 'BETAPDID' : [2064]
`C7x86_64` PDID : [-12, 1683], 'BETAPDID' : [2065]
`C8x86_64` PDID : [-13, 2007], 'BETAPDID' : [2066]
`OL7x86_64` PDID : [-14, 1683], 'BETAPDID' : [2065]
`OL6i386` PDID : [-16, 1681], 'BETAPDID' : [2063]
`OL6x86_64` PDID : [-15, 1682], 'BETAPDID' : [2064]
`OL8x86_64` PDID : [-17, 2007], 'BETAPDID' : [2066]

@juliogonzalez these are the numbers we need